README.md

```diff
@@ -1,21 +1,21 @@
-[![Build status](https://github.com/git/git/workflows/CI/PR/badge.svg)](https://github.com/git/git/actions?query=branch%3Amaster+event%3Apush)
+[![Build status](https://github.com/git/git/workflows/CI/PR/badge.svg)](https://github.com/git/git/actions? query=branch%3Amaster+event%3Apush)
 
 Git - fast, scalable, distributed revision control system
 =========================================================
 
 Git is a fast, scalable, distributed revision control system with an
-unusually rich command set that provides both high-level operations
-and full access to internals.
+unusually rich command set that provides both high-level operations and low-level operations
+and full access to the internals.
 
-Git is an Open Source project covered by the GNU General Public
+Git is an Open Source project covered by the GNU General Public.
 License version 2 (some parts of it are under different licenses,
-compatible with the GPLv2). It was originally written by Linus
-Torvalds with help of a group of hackers around the net.
+compatible with the GPLv2). It was originally written by Linus.
+Linux Torvalds with the help of a group of hackers around the net.
 
 Please read the file [INSTALL][] for installation instructions.
 
 Many Git online resources are accessible from <https://git-scm.com/>
-including full documentation and Git related tools.
+including full documentation and Git-related tools.
```

Comment:
From what I understand, this is more a matter of taste, both are correct. Therefore I would go with "if it ain't broke, don't fix it" approach here.